umbra/run.sh

```diff
@@ -5,9 +5,24 @@ TRIES=3
 cat queries.sql | while read query; do
     sync
     echo 3 | sudo tee /proc/sys/vm/drop_caches
+    umbra/bin/server -createSSLFiles -certFile db/umbra.cert -keyFile db/umbra.pem -address 0.0.0.0 db/umbra.db &> umbra.log &
```

Comment:
We need to restart the server, otherwise, we cannot measure a cold run.

We validated our results with duckdb. Unfortunately, some of the order by clauses are under specified.

Reply:
> Unfortunately, some of the order by clauses are under specified

sorry, didn't get it. is there any issue we need to resolve?

Reply:
oh, I see. but of course you could add them temporarily for the testing purpose.